### ISSUE TITLE
Add option to reverse this check

### ISSUE THREAD

**Issue by baurmatt** [voxpupuli/puppet-lint-absolute_classname-check#17]
From README.md

> Including a class by a relative name might lead to unexpected results in Puppet 3. This plugin is not recommended for use with Puppet code that has dropped support for Puppet 3 (EOL 20161231).

I'd like to see an option to reverse this check meaning a warning if `::` is present but I'm not even sure if it is possible to configure a Puppet-Lint plugin.

The basic implementation is already done here -> https://github.com/syseleven/puppet-lint-absolute_classname-check/tree/feature/reverse

**Comment by bastelfreak** [comment]
oh yes, a reverse check would be awesome <3

**Comment by baurmatt** [comment]
Are you aware of a way to configure a puppet-lint plugin or do we have to maintain another repository/plugin? :(

**Comment by alexjfisher** [comment]
@baurmatt For Vox Pupuli projects, the gem needs to be added back here somewhere.
https://github.com/voxpupuli/modulesync_config/blob/56bd6f808de27f428b633c8baafd4ad59ec3a0c3/config_defaults.yml#L44

and configuration somewhere in https://github.com/voxpupuli/modulesync_config/blob/56bd6f808de27f428b633c8baafd4ad59ec3a0c3/moduleroot/Rakefile.erb#L12

**Comment by baurmatt** [comment]
@alexjfisher Sounds good, but I think we should first release that change to Rubygems ;) Can you do that? I'm happy to do the change in modulesync_config once this is done.

**Comment by alexjfisher** [comment]
Totally forgot about actually needing a release! Looks like travis is setup correctly. Just need a release PR to bump the version.
